### PR TITLE
Include Ganache private chains (5777) in EIP-155

### PR DESCRIPTION
**Pull request by rickmark**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by rickmark** [comment]
Not sure this is still valid, Ganache uses 1337/5777 which means the network id is common but not the same as the chain...

File: EIPS/eip-155.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ This would provide a way to send transactions that work on Ethereum without work
 | 5              | Goerli                                     |
 | 42             | Kovan                                      |
 | 1337           | Geth private chains (default)              |
+| 5777           | Ganache GUI private chains (default)       |
```
Comment (Pandapip1) [inline EIPS/eip-155.md:68]: The rationale for this change is unclear. Why is this necessary? Does Gananche not already implement this?

**Comment by SamWilsn** [comment]
We've discussed this change on today's EIPIP, and we will not be modifying EIP-155 to add this chain id.

EIPs are not the best place to maintain tables that are intended to change over time.
